fixtures/axe-large-partial.js

```diff
@@ -0,0 +1,59 @@
+function getNode() {
+  return {
+    any: [
+      {
+        id: 'duplicate-id',
+        data: 'fixture',
+        result: true,
+        relatedNodes: []
+      }
+    ],
+    all: [],
+    none: [],
+    node: {
+      selector: [ '#fixture' ],
+      source: '<div id="fixture"></div>',
+      xpath: ['/div[@id="fixture"]'],
+      ancestry: ['html > body > div:nth-child(1)'],
+      nodeIndexes: [11]
+    }
+  };
+}
+
+const ruleResult = {
+  id: 'duplicate-id',
+  impact: 'serious',
+  pageLevel: false,
+  result: 'inapplicable',
+  nodes: []
+};
+for (let i = 0; i < 500_000; i++) {
```

Comment:
Tested this number if Playwright, Puppeteer, and Webdriverjs. Should be large enough to tigger each drivers size limit.